Makefile

```diff
@@ -209,7 +209,6 @@ proto-update-deps:
 	rsync -r --chmod 644 --include "*.proto" --include='*/' --exclude='*' $(TENDERMINT_PATH)/proto third_party
 	rsync -r --chmod 644 --include "*.proto" --include='*/' --exclude='*' $(COSMOS_SDK_PATH)/proto third_party
 	rsync -r --chmod 644 --include "*.proto" --include='*/' --exclude='*' $(IBC_GO_PATH)/proto third_party
-	rsync -r --chmod 644 --include "*.proto" --include='*/' --exclude='*' $(IBC_GO_PATH)/third_party/proto/confio third_party/proto
```

Comment:
Actually, think the proofs proto here just moved -- may just want to add a copy for https://github.com/cosmos/ibc-go/blob/main/third_party/proto/proofs.proto?  Believe IBC protos still depend on it

Reply:
Oh yeah nice find, i totally missed this. Yeah I'll make sure to include this file.

Reply:
https://github.com/Kava-Labs/kava/pull/1147/commits/a32e7af74efa5827b527a21b4bb9ef73d53a1025